test/utilities/backendValidation.test.tsx

```diff
@@ -59,6 +59,21 @@ const TESTING_PAIRS = [
     FOUND_ON_FORM: {},
     ERROR_MESSAGES: [{ field: 'Is Account Created', message: 'No' }],
   },
+  {
+    RESPONSE: [getString(6)],
+    FOUND_ON_FORM: {},
+    ERROR_MESSAGES: [{ field: '', message: getString(6) }],
+  },
+  {
+    RESPONSE: getString(7),
+    FOUND_ON_FORM: {},
+    ERROR_MESSAGES: [{ field: '', message: getString(7) }],
+  },
+  {
+    RESPONSE: null,
+    FOUND_ON_FORM: {},
+    ERROR_MESSAGES: [{ field: '', message: 'Response was null.' }],
```

Comment:
This is not a very user-friendly message. I'd go with the const `toastError` which is currently "Error submitting form" but can be improved to be even more friendly in the future!

Reply:
Like "There was an error submitting the form." would be neat and profesional 😮